bitrise.yml

```diff
@@ -1417,7 +1419,6 @@ workflows:
         - scheme: Fennec_Enterprise
         - configuration: Fennec_Enterprise
         - xcconfig_content: ENABLE_DEBUG_DYLIB=NO
-    - deploy-to-bitrise-io@2.10.0: {}
```

Comment:
Can you leave this? is it fine to have several deploy steps, better to not loose artifacts created there by default